### PR TITLE
Dark Mode

### PR DESCRIPTION
**Pull request by jstucke**

### CODE REVIEW

**Comment by dorpvom** [comment]
pr_review_0
table has not-nice looking padding to the right

**Comment by dorpvom** [comment]
pr_review_1
The hyper links / references should be highlighted more prominently

**Comment by dorpvom** [comment]
pr_review_3
Was it a concious decision to give these tables a background? I'm not sure, but I think I'd prefer it not having the background differ from the rest of the "dark" back.

**Comment by jstucke** [comment]
> pr_review_0 table has not-nice looking padding to the right

fun fact: this problem was not introduced by this PR -- is was just invisible :sweat_smile:

**Comment by dorpvom** [comment]
pr_review_4
Probably tough, but it would surely be nice to make the pacman look more smooth on the dark background

**Comment by dorpvom** [comment]
pr_review_5
Labels are tough to read. Should be light.

**Comment by dorpvom** [comment]
pr_review_6
This is also impractical. The file name is not really readable. Either font would have to be dark or highlighting would have to be darker.

**Comment by jstucke** [comment]
> pr_review_1 The hyper links / references should be highlighted more prominently

it seems colored links were disabled in FACT at some point (independent of dark mode). Do we want to change that?

File: src/web_interface/static/js/fact_statistics.js [hunk 1]
```diff
@@ -41,98 +41,173 @@ var chart_options = {
...
-
-    document.getElementById(canvas_id).onclick = function(evt){
-        var points = any_chart.getElementsAtEvent(evt);
-        var label = any_chart.data.labels[points[0]._index];
-        if ((points[0] !== undefined) && (label != "rest"))
+function setLinks(canvasId, chart, link) {
+    document.getElementById(canvasId).onclick = (evt) => {
+        const points = chart.getElementsAtEvent(evt);
+        const label = chart.data.labels[points[0]._index];
+        if ((points[0] !== undefined) && (label !== "rest"))
```
[the remark this answers is not in the document]
Comment (jstucke) [reply src/web_interface/static/js/fact_statistics.js:62]: just a linter fix (unexpected type coercion)

File: src/web_interface/templates/home.html [hunk 2]
```diff
@@ -10,20 +10,27 @@
...
 {% endblock %}
 
 {% block body %}
     <div class="row justify-content-center mt-4">
 
-        <div class="p-0 m-2">
-            <center>
-                <img width="100%" height="auto" src="{{ url_for('static', filename='fact_logo.png') }}" style="max-width: 445px;">
+        <div class="col col-xs-12 col-sm-8 col-lg-6 col-xl-4 p-0 m-2 home-block">
+            <div>
```
[the remark this answers is not in the document]
Comment (jstucke) [reply src/web_interface/templates/home.html]: replaced the `<center>` block but is indeed not necessary -> removed

**Review by dorpvom — APPROVED** [review]

**Review by dorpvom — APPROVED** [review]

**Pull request merged**